static/app/components/events/contexts/contextCard.tsx

```diff
@@ -0,0 +1,132 @@
+import {Link} from 'react-router';
+import styled from '@emotion/styled';
+
+import {
+  getContextMeta,
+  getContextTitle,
+  getFormattedContextData,
+} from 'sentry/components/events/contexts/utils';
+import {AnnotatedTextErrors} from 'sentry/components/events/meta/annotatedText/annotatedTextErrors';
+import Panel from 'sentry/components/panels/panel';
+import {StructuredData} from 'sentry/components/structuredEventData';
+import {space} from 'sentry/styles/space';
+import type {Group, Project} from 'sentry/types';
+import type {Event} from 'sentry/types/event';
+import {defined, objectIsEmpty} from 'sentry/utils';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface ContextCardProps {
+  alias: string;
+  event: Event;
+  type: string;
+  group?: Group;
+  project?: Project;
+  value?: Record<string, any>;
+}
+
+function ContextCard({alias, event, type, project, value = {}}: ContextCardProps) {
+  const organization = useOrganization();
+  if (objectIsEmpty(value)) {
+    return null;
+  }
+  const meta = getContextMeta(event, type);
+
+  const contextItems = getFormattedContextData({
+    event,
+    contextValue: value,
+    contextType: type,
+    organization,
+    project,
+  });
+
+  const content = contextItems.map(
+    ({key, subject, value: contextValue, action = {}}, i) => {
+      if (key === 'type') {
+        return null;
+      }
+      const contextMeta = meta?.[key];
+      const contextErrors = contextMeta?.['']?.err ?? [];
+      const hasErrors = contextErrors.length > 0;
+
+      const dataComponent = (
+        <StructuredData
+          value={contextValue}
+          depth={0}
+          maxDefaultDepth={0}
+          meta={contextMeta}
+          config={{}}
```

Comment:
nit: no need to pass `config`, it's not required

Reply:
It's typed as 
```
config: StructedEventDataConfig | undefined
```
 so its required to declare it. I'll update the type to:
```
config?: StructuredEventDataConfig
```
so that I don't have to set it here, and nothing should change.

Reply:
Sorry, I didn't realize that inner component was being exported. I added the `| undefined` because it's easy to forget one of the props when you are doing a recursive render. But for top-level components it's nice to keep the API minimal